src/controllers/userController.js

```diff
@@ -123,8 +125,87 @@ class UserController {
       });
     }
   }
+
+  /**
+   * @description - This method is responsible for assigning roles to users
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   *
+   * @returns {object} - object representing response message
+   *
+   * @memberof UserController
+   */
+  static async assignRole(request, response) {
+    try {
+      const { email, role } = request.body;
+      if (role !== 'admin' && role !== 'user') {
+        return response.status(400).json({
+          status: 'Fail',
+          message: 'Status can only be "admin" or "user"'
+        });
+      }
+      const fieldToUpdate = {
+        role
+      };
+      const updatedUser = await User.update(fieldToUpdate, {
```

Comment:
Hi @okoroemeka, I like your thought process here, but couldn't it have been more straightforward
if you just kept the variable 'role' with the curly braces inside the update method instead of having to declare another constant for it again? what do you think?

Reply:
I think this way is cleaner and easy for anyone to follow.